frontend/src/components/VDetailPerson.vue

```diff
@@ -2,61 +2,67 @@
   <div>
         <b-list-group>
           <b-list-group-item class="header-item active">
-            <icon class="icon-color" name="user"/>
-            Hans Pedersen
+            <icon class="mb-1" name="user"/>
+            {{result.name}}
           </b-list-group-item>
         </b-list-group>
 
-    <div class="card mt-2">
+    <div v-if="result.locations.length" class="card mt-2 mb-2">
       <div class="card-body">
-          <b-list-group-item variant="dark">
-            Adresse
-          </b-list-group-item>
-          <b-list-group-item>
-            <icon name="map-marker-alt"/>
-            <span class="col">Hold-an vej 7</span>
-          </b-list-group-item>
-          <b-list-group-item>
-            <icon name="phone"/>
-            <span class="col">34233350</span>
-          </b-list-group-item>
-          <b-list-group-item>
-            <icon name="envelope"/>
-            <span class="col">mila@gmail.dk</span>
-          </b-list-group-item>
+        <b-list-group-item variant="dark">
+          {{ $t('contact_info') }}
+        </b-list-group-item>
+        <b-list-group-item v-if="result.locations && result.locations.length" v-for="(location, idx) in result.locations" :key="Object.keys(location)[idx]">
```

Comment:
This looks awfully similar to `VDetailOrganisation` I think you could make a component for all this repetition. Or is there something I'm not seeing?